src/components/RequestBox/RequestBox.js

```diff
@@ -83,7 +93,14 @@ export default class RequestBox extends Component {
   };
 
   updateStateElement = (elementName, text) => {
-    this.setState({ [elementName]: text });
+    if (elementName == "patient") {
```

Comment:
this isn't a very react friendly way to accomplish this, what we want here seems to be to listen for changes to the patient state, which can be accomplished with [componentDidUpdate](https://legacy.reactjs.org/docs/react-component.html#componentdidupdate).  Though we could also rewrite everything to use react hooks.  But who could say which of those options would be easier.

Reply:
fixed.